app/facades/food_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodFacade
+  class << self
+    def foods_by_ingredient(ingredient)
+      results = FoodService.search_by_ingredient(ingredient)
+      Search.new(results)
```

Comment:
Essentially, your `Search` object is your Presenter. It's a good pattern that we haven't taught.